drivers/storage/portworx/portworx_test.go

```diff
@@ -7037,6 +7042,12 @@ func (m *fakeManifest) GetVersions(
 			AlertManager:              "quay.io/prometheus/alertmanager:v1.2.3",
 		},
 	}
+	if m.k8sVersion != nil && m.k8sVersion.GreaterThanOrEqual(k8sutil.K8sVer1_22) {
+		version.Components.PrometheusOperator = "quay.io/prometheus-operator/prometheus-operator:v1.2.3"
```

Comment:
Why not overwrite the other prometheus images too? That way we will have tested that other prometheus images also changed with the k8s version.